Seasonal Spotlights: Summer 2019 newspost (#2619)
ppy/osu-wiki: news/2019-09-30-beatmap-spotlights-summer-2019.md

---
layout: post
title: "Beatmap Spotlights: Summer 2019"
date: 2019-09-30 10:00:00 +0000
---

> Even if the hot season flew away and colder winds are blowing, we still have something in our pocket to keep you warm. Our selection from the hotest season is here for you the spend some time with. Come take a look and earn a fancy new medal for yourself by playing them all!

Reviewer: ```suggestion
Summer's already gone and the colder winds are blowing, but we still have something in our pocket to keep you warm! Our selection of maps from the hottest season is here for you the spend some time with. Come take a look and earn a fancy new medal for yourself by playing them all!
```

![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/header.jpg)

> The beatmap charts for the Summer Spotlights can be found [here](https://osu.ppy.sh/rankings/taiko/charts?spotlight=267). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Summer 2019 Spotlights rankings will end on the 15th of December, 2019.**

Reviewer: ```suggestion
The [ranking charts for Summer Spotlights](https://osu.ppy.sh/rankings/osu/charts?spotlight=267) are live! Their respective packs are available at the [Beatmap Pack listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Summer 2019 Spotlights rankings will end on the 15th of December, 2019.**
```

## Spring Spotlights results

> First of all, congratulations to the winners of the Seasonal Spotlights: Summer 2019. They will receive 3, 2, and 1 month of supporter for 1st, 2nd, and 3rd place respectively for their efforts!

Reviewer: ```suggestion
First of all, congratulations to the winners of the Spring 2019 Seasonal Spotlights. They will receive 3, 2, and 1 month(s) of supporter for 1st, 2nd, and 3rd place respectively for their achievement!
```

- **osu!:** [Toy](https://osu.ppy.sh/users/2757689), [Woey](https://osu.ppy.sh/users/3792472) and [xxluizxx47](https://osu.ppy.sh/users/4687701)
- **osu!taiko:** [Jaye](https://osu.ppy.sh/users/4841352), [AlvaroYL](https://osu.ppy.sh/users/6333166) and [Whulf](https://osu.ppy.sh/users/11601107)
- **osu!catch:** [RAMPAGE88](https://osu.ppy.sh/users/448547), [Predominador](https://osu.ppy.sh/users/4568537) and [Chatie](https://osu.ppy.sh/users/6524765)
- **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [Stink God](https://osu.ppy.sh/users/7381289) and [Cryolien](https://osu.ppy.sh/users/1626983)

> 

Reviewer: ```suggestion
```

> # Navigation

Reviewer: ```suggestion
## Navigation
```

> - <a href="#osu">osu!</a>

Reviewer: ```suggestion
- [osu!](#osu)
```
same with other 3 mode

- <a href="#taiko">osu!taiko</a>
- <a href="#catch">osu!catch</a>
- <a href="#mania">osu!mania</a>

> # Summer 2019 Spotlights

Reviewer: ```suggestion
# Summer Spotlights
```
to be consistent with other header

## <a name="osu" id="osu"></a>osu!

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/hide.jpg)](https://osu.ppy.sh/beatmapsets/972932)

What better way to introduce a new Featured Artist's music than with [Mirash](https://osu.ppy.sh/users/2841009)'s set of "Hide And Seek". This wonderfully jazzy piano track is all "get-up-and-go" featuring flurries of unconventional cut-stream patterns in the top difficulty. 

Lots of rhythmic variation allow both Mirash and [RLC](https://osu.ppy.sh/users/1047883) to play around with their patterning throughout the set's difficulty spread, leaving the player with a highly diverse set of levels to play through. All of this is complimented beautifully by [Noffy](https://osu.ppy.sh/users/1541323)'s exquisite storyboard, earning this map the top spot in this season's spotlights.

> _written by [Toy](https://osu.ppy.sh/users/2757689)_

Reviewer: move these to the beginning of each description, instead of the end
also use `*` instead of `_`

Author: I think that it is fine at the bottom, that way it stays consistent with our previous article

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/little.jpg)](https://osu.ppy.sh/beatmapsets/992322)

[Senseabel](https://osu.ppy.sh/users/6184386)'s interpretation of "MGMT - Little Dark Age" manages to create a very funky experience for both pro and amateur players alike. Movement throughout the map is very fluid, and fits the song so well that you'll find it hard to resist cursor-dancing. Simple, yet engaging rhythm fits every element of the song, giving you immersive gameplay. Complete with impressive hitsounding, you won't want to miss out on playing this one.

_written by [squirrelpascals](https://osu.ppy.sh/users/6151332)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/overtaken.jpg)](https://osu.ppy.sh/beatmapsets/898881)

Dear [Rue](https://osu.ppy.sh/users/417551) is back, after disappearing from the mapping scene for about 5 years! Their new mapset, "Reid Speed & Ahee - Overtaken (feat. Heartwurkz)" proved again why flow-oriented mapping would never become old-fashioned! 

> Mixing of various kinds of aesthetic design ideas, the top difficulty raised the definition of "look" to a whole new level - all of the different styles just compromise so well, yet doesn't ruin even a modicum of playability! Such a breath-taking and mind-blowing work is definitely worth checking and playing, and don't forget this modern masterpiece was made by a mapper who is supposed to be straight out of the ark!

Reviewer: ```suggestion
Mixing of various kinds of aesthetic design ideas, the top difficulty raised the definition of "look" to a whole new level⁠—all of the different styles compliment the map so well, and they don't sacrifice playability! Such a breath-taking and mind-blowing work is definitely worth checking and playing, and don't forget this modern masterpiece was made by a mapper who is supposed to be straight out of the ark!
```

_written by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/pray.jpg)](https://osu.ppy.sh/beatmapsets/940111)

[Mirash](https://osu.ppy.sh/users/2841009) and [Kalibe](https://osu.ppy.sh/users/3376777) combine their unique styles on the heartrending ending of Satsuriku no Tenshi to produce a wonderfully emotional map.

Kalibe's unique movement and rhythm combine with Mirash's distinctive approach to circle patterns in a way that fits this song incredibly well, creating an intensity that doesn't overwhelm the song but engages the player.

Be sure to check out the Insane collaboration between [Hey lululu](https://osu.ppy.sh/users/4086497) and [Garden](https://osu.ppy.sh/users/2849992) as well, it's absolutely lovely in its own right. And of course, both diffs are complemented by a very technically impressive storyboard by [Peter](https://osu.ppy.sh/users/8623835).

_written by [Deca](https://osu.ppy.sh/users/9088487)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/loser.jpg)](https://osu.ppy.sh/beatmapsets/852544)

> Out from last year’s Pending Cup comes Yonezu Kenshi’s "LOSER". This mapset includes six of the entries (plus a hard difficulty by [yf_bmp](https://osu.ppy.sh/users/1243669)), which all has unique styles that led to high rankings in the mapping contest. Highlights include [Yugu](https://osu.ppy.sh/u/3161834)’s grid-based mapping, [AIR](https://osu.ppy.sh/u/2070688)’s unique sliders and flow, and [how2miss](https://osu.ppy.sh/u/4477199)’ clean patterns and hitsounding.

Reviewer: ```suggestion
Out from last year’s Pending Cup comes Yonezu Kenshi’s "LOSER". This mapset includes six of the entries (plus a hard difficulty by [yf_bmp](https://osu.ppy.sh/users/1243669)), which all have unique styles that led to high rankings in the mapping contest. Highlights include [Yugu](https://osu.ppy.sh/users/3161834)’s grid-based mapping, [AIR](https://osu.ppy.sh/users/2070688)’s unique sliders and flow, and [how2miss](https://osu.ppy.sh/users/4477199)’ clean patterns and hitsounding.
```

The marathon itself is quite a simple difficulty, with clear visual, flow and patterns, based on EvilElvis older mapping, but he combination of slow pacing of the music and the main guitar leads are greatly represent by slider velocity and spacing changes, making this one of the best hard diffs to come out this year. For both casual, experience and even Higurashi fans out there, you should definetly try this map.

_written by [Net0](https://osu.ppy.sh/users/5099768)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/nejimaku.jpg)](https://osu.ppy.sh/beatmapsets/904067)

To truly evoke the sentiments tieLeaf's Nejimaku Tokei ga Tsuki no Michikake o Kizamu 2016ver. offers would demand intricate efforts for how vivid and lovely the track is, yet [ImpurePug](https://osu.ppy.sh/users/3124248) and their guest contributor [Mentai](https://osu.ppy.sh/users/8554786) both manage to exceed our expectations with one of the very impressive mapsets of this season.

> From ImpurePug's pattern-oriented and highly pristine style in the top difficulty which articulates the aestheticism of the track, to Mentai's strikingly raw interpretation which exceptionally captures its melodic and emotional wildness. Not to mention its charming hitsounding accompanied with a minimalist yet pleasant storyboard by ImpurePug themself with help from [Enkrypton](https://osu.ppy.sh/users/5178122). This mapset is bound to construct an experience that immerses you into the reverie of a flowery tale and a timeless fantasy.

Reviewer 2: ```suggestion
From ImpurePug's pattern-oriented and highly pristine style in the top difficulty which articulates the aestheticism of the track, to Mentai's strikingly raw interpretation which exceptionally captures its melodic and emotional wildness. Not to mention its charming hitsounding accompanied with a minimalist yet pleasant storyboard by ImpurePug themself with help from [Enkrypton](https://osu.ppy.sh/users/5178122). This mapset is bound to deliver an experience that immerses you into the reverie of a flowery tale and a timeless fantasy.
```

_written by [KnightC0re](https://osu.ppy.sh/users/7894340)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/radiate.jpg)](https://osu.ppy.sh/beatmapsets/967719)

Aggression defines this mapset, both through the harsh vocals and the even harsher patterns littered around the song's climaxes. The song begins boldly by claiming, "To take away our expression, is to impoverish our existence". [Hectic](https://osu.ppy.sh/users/6885942)(h4d0uk3n1) masterfully parallels the song's intense motif and sound through a mixture of high-speed straight sliders, and very linear jump patterned sections. 

The wide angles and linear patterning require jarring cursor movements, giving this map a true feel like you're playing to the music. [Senseabel](https://osu.ppy.sh/users/6184386) gives a similar, but unique take on the song using creative slider shapes, velocities, and tight patterning to give their difficulty the same jerky movement while keeping the map's visuals in-tact.

_written by [Toy](https://osu.ppy.sh/users/2757689)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/kawaki.jpg)](https://osu.ppy.sh/beatmapsets/924551)

> From countless sets of Minami's "Kawaki wo Ameku", Karen's map stands out with its diversity, every difficulty have a different feeling to offer.

Reviewer: ```suggestion
From countless sets of Minami's "Kawaki wo Ameku", [Karen](https://osu.ppy.sh/users/3143784)'s map stands out with its diversity, every difficulty having a different feeling to offer.
```

> Moecho's Unraveling Rain difficulty attracts with its clear visuals, smooth movement and challenging 1/4 patterns. That all, mixed with an amazing storyboard and impressive keysounding playing well-known Tokyo Ghoul opening - "unravel", creates an unforgetable and unique gameplay experience. Make sure to check it out!

Reviewer: ```suggestion
[Moecho](https://osu.ppy.sh/users/5075660)'s Unraveling Rain difficulty attracts with its clear visuals, smooth movement and challenging 1/4 patterns. All of that mixed with an amazing storyboard and impressive keysounding of the well-known Tokyo Ghoul opening, "unravel", creates an unforgettable and unique gameplay experience. Make sure to check it out!
```

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/stand.jpg)](https://osu.ppy.sh/beatmapsets/987998)

[hypercyte](https://osu.ppy.sh/users/9155377)'s map of the "peggies - Stand By Me" manages to provide a very unique experience that you won't find in many other ranked maps out there. Intuitive slider shapes give it a very visually interesting vibe, and use of unconventional movement throughout keeps the map exciting, while keeping you on your toes. 

These two things combine to give you a very refreshing experience, to go along with an inspiring song as well. It's a map that's obviously meant to be played, and not just talked about, so what are you waiting for? Go play it, it's cute!

_written by [squirrelpascals](https://osu.ppy.sh/users/6151332)_

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/osu/triplaneta.jpg)](https://osu.ppy.sh/beatmapsets/1010927)

[deetz](https://osu.ppy.sh/users/3513559) returns once again with their masterfully delivered rendition of "Triplaneta" by ELECTROCUTICA feat. F9, a 6-minute moment of pure bliss as the mapper demonstrates their prowess on a whole new level.

> A airy skip and a chill-paced dance, this map is a fantastic example of blending map design into the motif in music, with tasteful rhythm and patterning that give accentuation to momentum changes and a more freeform take on movement, culminating into a cohesive and euphoric experience you would not want to miss out.

Reviewer 2: ```suggestion
An airy skip and a chill-paced dance, this map is a fantastic example of blending map design into the motif in music, with tasteful rhythm and patterning that give accentuation to momentum changes and a more freeform take on movement, culminating into a cohesive and euphoric experience you would not want to miss out.
```

_written by [F D Flourite](https://osu.ppy.sh/users/2459589)_

## <a name="mania" id="mania"></a>osu!mania

[![](/wiki/shared/news/2019-09-30-beatmap-spotlights-summer-2019/mania/ari.jpg)](https://osu.ppy.sh/beatmapsets/933455)

One order of a venti frappuccino coming right up. [-MysticEyes](https://osu.ppy.sh/users/6253266) comes in with a bang with his map of "ari venti w/ Young Citrus"! 

The map is flavourful of many different patterns. Every sound is represented throughout the entire map giving it's unique concept. Even the long note length is taken into consideration to correspond to the sounds. If anything, the mapset is incredibly delicate and precise given its strict structure; long note lengths progressively following the song as well as the vocals, and fairly comfortable patterns for the hats when all comes together in the kiais. 

Mystic definitely outdone himself with this work of art. Song choice accompanies the structure beautifully with its light percussion and vocals.

> Oh, and here's your venti frappuccino - thank u, next.

Reviewer: ```suggestion

Oh, and here's your venti frappuccino—thank u, next.
```

Technical patterns, most of the time combined with LNs, can be spotted throughout the map, making it exceedingly difficult even for high-level players. Curiossity has proven time and again that he knows exactly how to craft a map that keeps people invested throughout its duration, and we hope that he will bring more maps like this to the ranked section in the near future!

_written by [\-mint\-](https://osu.ppy.sh/users/8976576)_

---

The spreadsheet is available for everyone who is interested to take a look at how we selected these mapsets. Do not be afraid to take a look at it [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vS4e7xNtDIFjlYurzJuNIwBiknKxG5QWsHk4b7wUandAdHL6I7IY_Ov6SR6EAlKE7SeIesjeAdinrho/pubhtml)!

We hope you enjoy our selection for this season and look forward to what we will bring to you in the future. Until then, happy circle clicking, drum bashing, fruit catching, and key smashing! Do not forget to have a rest occasionally!

—the Spotlights Team

> 

Reviewer: some extra newlines down here

### Navigation

- [osu!](#osu)

> - [taiko](osu!taiko)

Reviewer: backwards and missing #

Author: woops
